### ISSUE TITLE
Can't edit GPIO interface

### ISSUE THREAD

**Issue by Geoff46** [WebThingsIO/gpio-adapter#36]
I've just started using the Mozilla-iot gateway to control the GPIOs on a Raspberry Pi.
I've successfully set it up to turn a pin on and off. However I want to change the name I originally gave the button when setting it up. I have changed the name in the GPIO configuration screen but when I go back to the control screen it still has the old name. I've tried re-scanning but it still doesn't update to the new name.

I tried adding another button in the GPIO configuration with the new name but it doesn't appear in the control screen.
I deleted the original button from the GPIO configuration but it still appears as it was.
I tried removing the GPIO addon completely and then re-installing it. 
I've rebooted the Pi and restarted the iot web interface

I'm getting nowhere - whatever I do the interface has permanently set itself on what I originally added. I can't change it, delete the old, or add any new, button to control the GPIO.
I'm stumped!
Grateful for any pointers as to where to go next.

**Comment by mrstegeman** [comment]
To rename a device, you'll want to do the following:
1. Click the spider icon on your device (on the main things screen) to open the properties view.
2. Click the three-dot menu in the lower right, then click Edit.
3. Change the device's name and click save.

**Comment by Geoff46** [comment]
Thanks for your reply. After some frustration I managed to follow your instructions! I was able to change the name successfully - the process is not at all intuitive and without your help I would never have done it. 
Logically, editing the name and parameters in the GPIO setup menu and applying the changes would be the way to do it.
I'll play around now and see if I can get any other changes to work - e.g. adding new buttons, deleting buttons.
That little 3-dot menu is well hidden unless you know where to look!

Thanks again, I feel a little less stumped now!!

Regards
